infra_monitoring_account/tests/unit/test_infra_monitoring_stack.py

```diff
@@ -1,7 +1,7 @@
 import aws_cdk as core
 import aws_cdk.assertions as assertions
 
-from infra_monitoring.infra_monitoring_stack import InfraMonitoringStack
+from infra_monitoring_account.infra_monitoring_stack import InfraMonitoringStack
```

Comment:
Maybe, it'd be better to change name to "infra_monitorED_account" (both, folder and stack names) - as we discussed earlier.

Reply:
Fixed